functions/db/community.js

```diff
@@ -14,4 +14,32 @@ const getCommunityPostDetail = async (client, communityPostId) => {
   return convertSnakeToCamel.keysToCamel(rows[0]);
 };
 
-module.exports = { getCommunityPostDetail };
+const getCommunityPosts = async (client, limit, page) => {
+  const { rows } = await client.query(
+    `
+    SELECT cp.id, u.nickname, cp.title, cp.body, cp.content_url, cp.content_title, cp.content_description, cp.thumbnail_url, cp.created_at
+    FROM community_post cp
+    JOIN "user" u on cp.user_id = u.id
+    WHERE cp.is_deleted = FALSE
+    ORDER BY cp.created_at DESC
```

Comment:
```suggestion
    ORDER BY cp.created_at DESC, cp.id DESC
```
이거 offset 기반 페이지네이션에서 꽤 유명한 문제인데  order by column 이 중복될 경우 여러 페이지에 중복 값이 나올 경우의 수가 있습니다.
그래서 해결법으로 pk 같이 고유한 인덱스로 한번 더 정렬을 해줍니다.

Reply:
사실 게시판 형태가 아니고 무한스크롤이라 추후에 성능 딸리면 cursor 기반 고려해보면 좋을 것 같아요

Reply:
고유 column 정렬로 중복 값 문제가 해결이 되는군요..!!

나중에 데이터가 쌓였을 때 커서 기반으로 페이지네이션을 적용해 개선해보겠습니다 :)